Ladybird/Headless/Test.cpp

```diff
@@ -44,11 +101,33 @@ static ErrorOr<void> load_test_config(StringView test_root_path)
     }
 
     auto config = config_or_error.release_value();
+    auto current_oses = current_os_list();
 
     for (auto const& group : config->groups()) {
         if (group == "Skipped"sv) {
-            for (auto& key : config->keys(group))
-                s_skipped_tests.append(LexicalPath::join(test_root_path, key).string());
+            for (auto& key : config->keys(group)) {
+                auto value = config->read_entry(group, key);
+
+                // Suppress on all platforms by default
+                if (value.is_empty()) {
+                    dbgln(" Skipping test '{}'", key);
```

Comment:
I think this `dbgln` and the one below will add a bit too much noise every time we run these tests. We already print skipped tests at the end of the run, we don't need to print them a second time here.

Reply:
Fair enough. I left them in by mistake at first, but then decided to leave them, so I'm happy to take them back out.